### DISCUSSION TITLE
Tabby themes

### DISCUSSION THREAD

**Discussion by MaxAlekseyev** [Eugeny/tabby#8057] [Q&A]
Hello,

The Tabby have a lot of great themes but I can't understand how to apply them. Diffent terminals have one BW theme.
For example Ubunty in Windows terminal has the fine colors but BW in tabby

![image](https://user-images.githubusercontent.com/10415908/223814814-1333cd7f-53bd-4011-9134-0b936e4e4a12.png)

![image](https://user-images.githubusercontent.com/10415908/223815055-aa2bb30e-a427-407b-817d-a8d3335b95b4.png)

**Comment by Eugeny** [comment]
https://github.com/Eugeny/tabby/discussions/5258
